### PR TITLE
Cleanup when update dialogs show

### PR DESCRIPTION
**Pull request by jonsmorrow**
Don't show update not available dialog when changing update channel.

Signed-off-by: Jon Morrow <jmorrow@chef.io>

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -78,8 +79,8 @@ function createTray() {
   tray.setContextMenu(trayMenu);
 }
 
-function triggerUpdateCheck(requestFromUser=false) {
-  app.emit('do-update-check', requestFromUser);
+function triggerUpdateCheck(requestFromUser=false, displayUpdateNotAvailableDialog=true) {
+  app.emit('do-update-check', requestFromUser, displayUpdateNotAvailableDialog);
```
Comment (marcparadise) [inline main.js]: I don't think  we need both of these variables, since they're both used for determining whether to display the update dialog.

**Comment by jonsmorrow** [comment]
Yeah happy to discuss this one.

**Comment by jonsmorrow** [comment]
In testing it seemed we should have something more than just the icon to let the user know on startup.

**Comment by jonsmorrow** [comment]
Wouldn't want the no updates to show and we had no way to differentiate before. Anyway this is out of scope for ga.

File: main.js (hunk 1 above)
Comment (jonsmorrow) [reply main.js]: The first decides wether to display any dialogs. The second decides if the no update dialog is presented when we are showing dialogs.

**Pull request merged**